app/views/maily/shared/_sidebar.html.erb

```diff
@@ -3,10 +3,20 @@
     <section class="nav_list">
       <h3 class="nav_title"><%= "#{mailer.name.humanize} (#{mailer.total_emails})" %></h3>
       <ul>
+        <% email_name_list = [] %>
         <% mailer.emails_list.each do |email| %>
-          <li><%= link_to email.name.humanize, maily_email_path(mailer: mailer.name, email: email.name), class: sidebar_class(mailer, email) %></li>
+          <% next if email_name_list.include?(email.name) %>
+          <% email_name_list << email.name %>
```

Comment:
instead of having this logic in the html, I'd move it to a helper, something like `a_helper_method(mailer.emails_list).each` where we can filter out those passed emails